docs/advanced/TPP-Snap.md

```diff
@@ -1,10 +1,10 @@
 # TPP-Snap
 
-> TODO: Add some general information about TTP-Snap and explain how it is used inside of the pwa (image of the architecture?)
+TPP-Snap is a Javascript library designed to connect the FirstSpirit ContentCreator UI with dynamically generated HTML elements (like in a PWA). This connection is achieved using preview ids. When you add a known preview id as an attribute to an HTML element the TPP-Snap library will automatically decorate it using borders and buttons when your PWA is viewed in the FirstSpirit ContentCreator. The library also offers the tools to add your own buttons as you see fit.
```

Comment:
"TPP" is an internal/ technical name only... I think it should be "OmnichannelManager" here (and in the link from the index.md).

A link to https://docs.e-spirit.com/tpp/index.html.en would also be good.

"will automatically decorate it using borders" is not enough, in my opinion. I think it needs to be clearer that this allows editing of elements (maybe also mentioning workflows etc).